pages/translations/ar/LC_MESSAGES/messages.po

```diff
@@ -1581,7 +1581,7 @@ msgid "Prioritize resource loading"
 msgstr ""
 
 #: pages/content/amp-dev/about/how-amp-works.html:85
-msgid "AMP controls all resource downloads  it prioritizes resource loading, loading only what’s needed, and prefetches lazy-loaded resources."
+msgid "AMP controls all resource downloads: it prioritizes resource loading, loading only what’s needed, and prefetches lazy-loaded resources."
```

Comment:
This sentence is also awful. It says uses the word `resources` and `loading` three times.

Reply:
😬